### DISCUSSION TITLE
how to connect using SSH?

### DISCUSSION THREAD

**Discussion by ghost** [aio-libs/aiomysql#813] [Q&A] [answered]
### Describe the bug

tell me how to connect with SSH
I SEE SSL . But  ssh I do not see

### To Reproduce

.

### Expected behavior

.

### Logs/tracebacks

```python-traceback
.
```

### Python Version

```console
$ python --version
```

### aiomysql Version

```console
$ python -m pip show aiomysql
```

### PyMySQL Version

```console
$ python -m pip show PyMySQL
```

### SQLAlchemy Version

```console
$ python -m pip show sqlalchemy
```

### OS

Window 

### Database type and version

```console
SELECT VERSION();
```

### Additional context

tell me how to connect with SSH
I SEE SSL . But  ssh I do not see

### Code of Conduct

- [X] I agree to follow the aio-libs Code of Conduct

**Comment by ghost** [comment]
Help me

**Comment by Nothing4You** [comment, marked as answer]
Hi,

I don't know what you mean.
I'm not aware of any SSH transport for MySQL.

You can of course run SSH tunneling separately to forward ports or sockets, but that functionality is not included in aiomysql.
See https://man.openbsd.org/ssh#L
